### ISSUE TITLE
Allow a delay between successive API calls

### ISSUE THREAD

**Issue by malminhas** [scanapi/scanapi#266]
### Enable the possibility to `sleep` between API calls.  

The API I am accessing for test purposes builds up a list of quotes returning a status of `PROGRESSING` until all quotes are retrieved at which point the status is `COMPLETED`.  In other words, this API has to be polled.  I am able to work around this with a sequence of around 5 identical calls until I get past the point of completion.  It would be helpful to have the ability to introduce a delay between successive calls to allow this to be done more compactly perhaps something like this where `delay` is a `sleep` time? 

```
      - name: get_quotelist
         path: v2/quotes/${quotelist_id}
         method: get
         body:
         headers:
           Content-Type: "application/json"
           Accept: "application/json"
           Authorization: ${{ "Bearer ${token}" }}
         tests:
          - name: status_code_is_200
            assert: ${{ response.status_code == 200 }}
          - name: has_quote_items
            assert: ${{ len(response.json().get("quotes")) > 0 }}
          - name: status_progressing
            assert: ${{ response.json().get("status") in ["PROGRESSING", "COMPLETED"] }}
         delay: 1
```

**Comment by kelvins** [comment]
Can I work on this feature?
What do you think about including a `delay` key at the `endpoints` level and applying it to every request that is inside the endpoint?

**Comment by malminhas** [comment]
Thanks @kelvins thanks for offering to help!  I think your suggestion sounds quite reasonable.  I'd be happy to test.  I also think the delay is best set in milliseconds.  At least that would be most useful in the particular case I have.

**Comment by camilamaia** [comment]
@malminhas this feature is already available for you on version v2.1.0 🎉🎉🎉 
https://pypi.org/project/scanapi/

Thanks @kelvins for developing it 🙇‍♀️
